schema/rpc.fbs

```diff
@@ -160,6 +163,9 @@ table SerialUpdateResponse {
     log: string;
     closed: bool;
 }
+table SerialTrackerRebootRequest {}
+table SerialTrackerGetInfoRequest {}
```

Comment:
Add comment (`///`) explaning what get info means

Reply:
Should be resolved